File: backend/Coboss.Types/DTO/ProjectDTO.cs
namespace Coboss.Types.DTO
{
    public class ProjectDTO
    {
        public string Number { get; set; } = default!;
        public string Name { get; set; } = default!;
        public string Description { get; set; } = default!;
        public DateTime Term { get; set; } = default!;
        public EmployeeDTO Manager { get; set; } = default!;
        public ICollection<BusinnessTaskDTO> BusinnessTasks { get; set; } = default!;
    }
}

File: frontend/src/types/DTO/ProjectDTO.ts
import BusinnessTaskDTO from "./BusinnessTaskDTO"
import EmployeeDTO from "./EmployeeDTO"

type ProjectDTO = {
  id: number
  number: string
  name: string
  description: string
  term: Date
  manager: EmployeeDTO
  businnessTasks: BusinnessTaskDTO[]
}

export default ProjectDTO